class/defaults.yml

```diff
@@ -1,3 +1,43 @@
 parameters:
   ingress_nginx:
     namespace: syn-ingress-nginx
+    release_name: ingress-nginx
+    charts:
+      ingress-nginx: '3.21.0'
+    helm_values:
+      controller:
+        config:
+          proxy-body-size: 1024m
+          proxy-buffer-size: 64k
+          log-format-upstream: >-
+            {
+            "time": "$time_iso8601",
+            "remote_addr": "$proxy_protocol_addr",
+            "x-forward-for": "$proxy_add_x_forwarded_for",
+            "req_id": "$req_id",
+            "remote_user": "$remote_user",
+            "bytes_sent": $bytes_sent,
+            "request_time": $request_time,
+            "status": "$status",
+            "host": "$host",
+            "request_proto": "$server_protocol",
+            "request_uri": "$uri",
+            "request_query": "$args",
+            "request_length": $request_length,
+            "request_time": $request_time,
+            "request_method": "$request_method",
+            "http_referer": "$http_referer",
+            "http_user_agent": "$http_user_agent",
+            "namespace": "$namespace",
+            "ingress_name": "$ingress_name",
+            "service_name": "$service_name",
+            "service_port": "$service_port"
+            }
+        publishService:
```

Comment:
Which type of service gets publisched here? I expect it to be of type `LoadBalancer`. This is good for most cases but not all. In case of AWS, the created AWS loadbalancer infront, could be of the wrong type (from the top of my head: `ALB` vs `NLB`). Does this data structure allow to set it accordingly (host port)?

Reply:
publishService is not equal to service, which defines in the helm chart, what you describe. The service is defined specifically, in the distribution part, because you sometimes need specific annotations in the LoadBalancer service. I will add the cloud specific configurations to the documentation, if we can agree on, that all helm chart values `helm_values` are over-writable.

Reply:
> […] if we can agree on, that all helm chart values helm_values are over-writable.

Yes this is the basic agreement (https://syn.tools/syn/explanations/jsonnet.html#_merging_external_config_data).

I am fine with having this covered in documentaion.